readme.md

```diff
@@ -1,5 +1,7 @@
 # Jetpack
 
+aa
```

Comment:
I think this was added by mistake.

Reply:
thanks for catching it! completely missed it myself